src/source/Ice/ConnectionListener.c

```diff
@@ -136,6 +154,7 @@ STATUS connectionListenerRemoveAllConnection(PConnectionListener pConnectionList
     BOOL locked = FALSE;
     PDoubleListNode pCurNode = NULL;
     PSocketConnection pSocketConnection = NULL;
+    const UINT64 longTimeout = 1 * HUNDREDS_OF_NANOS_IN_AN_HOUR;
```

Comment:
same - #define it

Reply:
Also, why is this function not calling connectionListenerRemoveConnection underneath it when iterating?

Reply:
then we have to wait for each connection to be closed before closing the next one.

Reply:
Got it